### ISSUE TITLE
Poor keyboard accessibility for paginated sections

### ISSUE THREAD

**Issue by samcf** [samcf/ogres#138]
The pagination component for paging through the token and scene image galleries cannot be accessed and operated with the keyboard and are skipped entirely by tabbing.

Sample of the paginated token gallery.
![Screenshot 2024-04-24 at 2 24 28 PM](https://github.com/samcf/ogres/assets/1691525/32647e5d-99ee-44d8-a6ae-c02d61e434ae)

These controls should be accessible with the keyboard, possibly implemented as a radio button group to utilize native browser shortcuts.

**Comment by samcf** [comment]
Resolved by 9ac0138
